layouts/shortcodes/semantic-color.md

```diff
@@ -0,0 +1,2 @@
+| Tag key                       | Description                                                                                                                                                                                                                                                                                                    |
+| :---------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
This file exists so that local build doesn't throw errors, it will get replaced with the real table during the ci/cd